client/packages/common/src/authentication/AuthContext.tsx

```diff
@@ -218,7 +218,7 @@ export const AuthProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
       store,
       token,
       user: {
-        id: '',
+        id: userDetails?.userId ?? '',
```

Comment:
An pre-existing oversight: the userId wasn't actually being returned in AuthContext.